packages/rpc-rx/src/types.d.ts

```diff
@@ -2,21 +2,23 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
-import { BehaviorSubject, Observable } from 'rxjs';
-
+import { ReplaySubject, Observable } from 'rxjs';
 import interfaces from '@polkadot/jsonrpc/types';
 
-export type RxRpcInterface$Method = (...params: Array<any>) => Observable<any> | BehaviorSubject<any>;
+export type RxRpcInterface$Method = (...params: Array<any>) => Observable<any> | ReplaySubject<any>;
 
 export type RxRpcInterface$Section = {
   [index: string]: RxRpcInterface$Method
 };
 
+export type RxRpcInterface$Events = 'connected' | 'disconnected';
```

Comment:
could we rename `RxRpcInterface$Events` to `RpcRxInterface$Events`, since we're referring to the class as `RpcRx` everywhere?

Reply:
As above.